test/functionalTest/cases/0787_cprs_full_functional/update_one_forward_in_ngsiv2.test

```diff
@@ -0,0 +1,267 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+One Forward Update To Context Provider
+
+--SHELL-INIT--
+dbInit CB
+dbInit CP1
+brokerStart CB
+brokerStart CP1
+
+--SHELL--
+
+#
+# 01. Update/APPEND E1/T1/A1 in CP1
+# 02. Register E1/T1/A1 in CB, provApp: CP1
+# 03. Update/UPDATE E1/T1/A1
+# 04. Query E1/T1/A1 in CB
+# 05. Query E1/T1/A1 in CP1
+#
+
+echo "01. Update/APPEND E1/T1/A1 in CP1"
+echo "================================="
+payload='{
+  "contextElements": [
+    {
+      "type": "T1",
+      "id":   "E1",
+      "attributes": [
+        {
+          "name": "A1",
+          "type": "string",
+          "value": "A1 in CP1"
+        }
+      ]
+    }
+  ],
+  "updateAction": "APPEND"
+}'
+orionCurl --url /v1/updateContext --payload "$payload" --port $CP1_PORT
+echo
+echo
+
+
+echo "02. Register E1/T1/A1 in CB, provApp: CP1"
+echo "========================================="
+payload='{
+  "contextRegistrations": [
+  {
+    "entities": [
+      {
+         "type": "T1",
+         "isPattern": "false",
+         "id": "E1"
+      }
+    ],
+    "attributes": [
+      {
+        "name": "A1",
+        "type": "string",
+        "isDomain": "false"
+      }
+    ],
+    "providingApplication": "http://localhost:'${CP1_PORT}'/v1"
+    }
+ ],
+ "duration": "P1M"
+}'
+orionCurl --url /v1/registry/registerContext --payload "$payload"
+echo
+echo
+
+
+echo "03. Update/UPDATE E1/T1/A1"
+echo "=========================="
+payload='{
+  "A1": {
+      "type": "string",
+      "value": "A1 via CB"
+  }
+}'
+orionCurl --url /v2/entities/E1/attrs?type=T1 --payload "$payload" -X PATCH
+echo
+echo
+
+
+echo "04. Query E1/T1/A1 in CB"
+echo "========================"
+payload='{
+  "entities": [
+    {
+      "id":   "E1",
+      "type": "T1"
+    }
+  ],
+  "attributes": [
+    "A1"
+  ]
+}'
+orionCurl --url /v1/queryContext --payload "$payload"
```

Comment:
In this test, step 03 (Update/UPDATE E1/T1/A1) is the critical part.
That is where v2 MUST be used.

Step 04 and 05 is just to ensure that step 03 did what it should